docs/OTSubscriber.md

```diff
@@ -83,6 +88,7 @@ class App extends Component {
     this.subscriberProperties = {
       subscribeToAudio: false,
       subscribeToVideo: true,
+      subscribeToCaptions: false,
```

Comment:
This value should be true, right? A subscriber will only start receiving captions by calling the subscribeToCaptions(true)

Reply:
This is just a code sample on how to use OTSubscriber properties. It is not a sample showing how to use captions.